calibrators/mapping_based_calibrator/README.md

```diff
@@ -0,0 +1,238 @@
+| -------------------------------- | ------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `calibration_camera_info_topics` | `sensor_msgs::msg::CameraInfo`                    | The intrinsic parameters of calibration cameras. `calibration_camera_info_topics` is provided via parameters (currently not used).  |
+| `calibration_image_topics`       | `sensor_msgs::msg::CompressedImage`               | Calibration cameras' image topics. `calibration_image_topics` is provided via parameters (currently not used).                      |
+| `mapping_pointcloud`             | `sensor_msgs::msg::PointCloud2`                   | Pointcloud's topic used for mapping. It is recommended to select the lidar that has the highest resolution and best FOV.            |
+| `calibration_pointcloud_topics`  | `sensor_msgs::msg::PointCloud2`                   | Pointclouds' topics used for calibrating with the `mapping pointcloud`. `calibration_pointcloud_topics` is provided via parameters. |
+| `detected_objects`               | `autoware_perception_msgs::msg::DetectedObjects`  | Messages containing detected objects, used in the filtering procedure.                                                              |
+| `predicted_objects`              | `autoware_perception_msgs::msg::PredictedObjects` | Messages that contain predicted object paths and positions, used in the filtering procedure.                                        |
+
+### Output
+
+| Name                              | Type                                   | Description                                                                                                    |
+| --------------------------------- | -------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| `output_map`                      | `sensor_msgs::msg::PointCloud2`        | Output map constructed from the `mapping_pointcloud`.                                                          |
+| `frame_path`                      | `nav_msgs::msg::Path`                  | The actual path of the `mapping_pointcloud`.                                                                   |
+| `frame_predicted_path`            | `nav_msgs::msg::Path`                  | The predicted path of the`mapping_pointcloud`.                                                                 |
+| `keyframe_path`                   | `nav_msgs::msg::Path`                  | The keyframe path of the `mapping_pointcloud`.                                                                 |
+| `keyframe_markers`                | `visualization_msgs::msg::MarkerArray` | Markers for keyframes.                                                                                         |
```

Comment:
Keyframe markers is ok as a description. It is sufficiently self explanatory
There are several missing `the` from here on in the table